packages/theme-chalk/src/common/var.scss

```diff
@@ -429,7 +429,7 @@ $select-group: () !default;
 $select-group: map.merge(
   (
     'text-color': getCssVar('color-info'),
-    'height': 30px,
+    'height': 34px,
```

Comment:
This seems to be a breaking change, maybe we should mark it somewhere. 🤔

Reply:
![image](https://github.com/element-plus/element-plus/assets/38392315/aa9302f4-765c-4702-8a23-9c11cf279fa7)
This only effect the group title, do you think it's a big change?

Reply:
I looked at it again and it seems to have no effect. The height set here is used in line 23 below, but the style set here is overwritten by the inline style.